skimage/_build_utils/tempita.py

```diff
@@ -17,10 +17,6 @@ def process_tempita(fromfile, outfile=None):
     E.g. processing `template.c.in` generates `template.c`.
 
     """
-    if outfile is None:
```

Comment:
I would consider also removing the default value for `outfile` argument.

Reply:
Yes, that should be fine. Just make sure that the behavior `tempita.py myfile.c.in` results in the output being written to `myfile.c` in the same directory.

Reply:
> I would consider also removing the default value for `outfile` argument.

@soupault looks like @lagru addressed this in 5ec1d3790e4492b705c5b7702f540623c9fd1aa6.